CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,9 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+== 2022-07-28 Clear Button in Database Importer
+<p>The "import from database" pane now has a clear button next to the query button. Clicking clear will clear the results in the currently selected RunPane and will clear the query text</p>
```

Comment:
Could you capitalise "Import", "Database", and the first "Clear" in the 2nd sentence (just to distinguish clear the button vs clear the function). Possibly also change "currently selected RunPane" to "current run" as the average user won't be familiar with a RunPane (the whatsnew entry is to inform the user rather than the developer)